.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,69 @@
+name: Bug Report
+description: Report a bug in OVHcloud Control Panel application
+title: '<title>'
+labels: ['bug']
+body:
+  - type: checkboxes
+    attributes:
+      label: Have you already contact our support center?
+      description: Before to open an issue on our GitHub repository, did you already contacted our support center? (See https://help.ovhcloud.com)
+      options:
+        - label: Yes, I have contact the support center.
+          required: true
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+        - label: I have searched the existing issues
+          required: true
+  - type: textarea
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps To Reproduce
+      description: Steps to reproduce the behavior.
+      placeholder: |
+        1. In this environment...
+        2. With this config...
+        3. Run '...'
+        4. See error...
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Expected Behavior
+      description: A concise description of what you expected to happen.
+    validations:
+      required: false
+  - type: dropdown
+    id: browsers
+    attributes:
+      label: What browsers are you seeing the problem on?
+      multiple: true
+      options:
+        - Firefox
+        - Chrome
+        - Safari
+        - Microsoft Edge
+        - Opera
+  - type: dropdown
+    id: device
+    attributes:
+      label: What device are you seeing the problem on?
+      options:
+        - Desktop
+        - Tablet
+        - Smartphone
+  - type: textarea
+    attributes:
+      label: Additional information to add?
+      description: |
+        Add any other context about the problem here. (Screenshots, links, etc…)
```

```suggestion
        Add any other relevant information about the problem here. (Screenshots, links, etc…)
```
Maybe we should also specify that information should be anonymized ? No data ?